### ISSUE TITLE
Soften our word choice "charge" since we are not making people pay yet

### ISSUE THREAD

**Issue by em-herrick** [GSA/notifications-admin#2043]
- [ ] Audit where in sending flow (e.g. you'll be charged for x messages" and on UX we talk about pricing/charing people.
- [ ] Provide places and alternative language to "charge"
- [ ] Remove/soften mentions of message "parts"

**Comment by ecayer** [comment]
I suggest that we roll this into message parts -> messages and paid-product content work. This is just an consideration @dmvancura  to ensure we keep both user groups, the paid and free tier members, in mind for the copy.

**Comment by dmvancura** [comment]
i will review the content for "charge" as well. thank you. (assigned to me now)

**Comment by dmvancura** [comment]
on the page "pricing/tracking usage/message parts." 'charge' is used 6 times. cathy and i are moving to archive that page and address pricing and characters separately at a later time (when the whole team agrees it's needed, because it'll be on a public-facing page).

**Comment by dmvancura** [comment]
found to counts of "charge" on the template creation userflow. attached are suggested copy updates.

![Image](https://github.com/user-attachments/assets/7a8d4376-0b5f-4112-8189-56248c5f2f71)
![Image](https://github.com/user-attachments/assets/dc7ebbf3-5a00-4b07-910a-6d4c2df27319)

**Comment by CathyBeil** [comment]
IMO we should remove the text under the template entirely, because once we make the transition, it will always say "will be charged as 1 text message," so it seems kind of unnecessary.

I'm indifferent about keeping something about message length under "how to customize." I suppose people might still wonder so it might be good to say something like "each message sent to each recipient is counted as one text message, view your dashboard to see how many messages you have used and have remaining."

**Comment by dmvancura** [comment]
agreed Cathy; when "part" transitions to "messages", we remove the text under the template. a count (e.g., 1) of any kind after that may confuse the user if we have it there

and the statement within the "message length" box could definitely be: "Each message sent to a recipient counts as one text. Keep track on your Dashboard of how many messages have sent and are remaining."

**Comment by dmvancura** [comment]
morning @alexjanousekGSA this is ready for next steps. action to take update Message Length copy in box to say: Each message sent to a recipient counts as one text. Keep track on your Dashboard of how many messages have sent and are remaining.

**Comment by dmvancura** [comment]
afternoon @CathyBeil FYI after viewing this multiple times, this seems related to message parts to messages, so i'm working with Alex to update the verbiage just within the Message length box

**Comment by dmvancura** [comment]
oh i just noticed this was closed, and it may have been indeed related to the ticket in which i reviewed, but this is also about text that show in a pre-error explanation on the create message template page. care to meet tomorrow about it @heyitsmebev or @alexjanousekGSA? unless team decided to close it. LMK. i can put 15mins on the calendar. thank you much
